config/celery-test-docker/00-main.yaml

```diff
@@ -50,25 +50,25 @@ scoring_systems:
     class: ores.scoring_systems.CeleryQueue
     score_cache: local_redis
     BROKER_URL: redis://localhost
-    BROKER_TRANSPORT_OPTIONS: {'socket_timeout': 15} # seconds
+    broker_transport_options: {'socket_timeout': 15} # seconds
     CELERY_RESULT_BACKEND: redis://localhost
-    CELERY_ACCEPT_CONTENT: ['pickle']
-    CELERY_TASK_SERIALIZER: 'pickle'
-    CELERY_RESULT_SERIALIZER: 'pickle'
+    accept_content: ['pickle']
+    task_serializer: 'pickle'
+    result_serializer: 'pickle'
     CELERYD_CONCURRENCY: 8
-    CELERYD_HIJACK_ROOT_LOGGER: false
+    worker_hijack_root_logger: false
     queue_maxsize: 100 # pending tasks
   docker_celery:
     class: ores.scoring_systems.CeleryQueue
     score_cache: docker_redis
-    BROKER_URL: redis://redis
-    BROKER_TRANSPORT_OPTIONS: {'socket_timeout': 15} # seconds
-    CELERY_RESULT_BACKEND: redis://redis
-    CELERY_ACCEPT_CONTENT: ['pickle']
-    CELERY_TASK_SERIALIZER: 'pickle'
-    CELERY_RESULT_SERIALIZER: 'pickle'
+    BROKER_URL: redis://localhost
+    broker_transport_options: {'socket_timeout': 15} # seconds
+    CELERY_RESULT_BACKEND: redis://localhost
```

Comment:
CELERY_RESULT_BACKEND --> result_backend

Reply:
You can find the list in http://docs.celeryproject.org/en/v4.1.0/whatsnew-4.0.html#v400-upgrade-settings and http://docs.celeryproject.org/en/v4.1.0/userguide/configuration.html#new-lowercase-settings